backend/homework/7/src/main/java/shrishti/example/Main.java

```diff
@@ -0,0 +1,40 @@
+package shrishti.example;
+
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import shrishti.example.beans.Vehicle;
+import shrishti.example.config.*;
+import shrishti.example.Logger.logBack;
+
+
+public class Main {
+    public static Vehicle expensiveVehicle(Vehicle v1, Vehicle v2){
+          if(v1.getCost() > v2.getCost()){
```

Comment:
Variable naming should denote the sense and logic it wishes to encapsulate.
Not plain v, v1, etc